src/main/java/com/cloudlabs/server/compute/ComputeController.java

```diff
@@ -59,11 +60,11 @@ public List<ComputeDTO> listComputeInstances() {
 		return response;
 	}
 
-	@GetMapping("/instance")
-	public ComputeDTO getComputeInstance(@RequestParam String instanceName)
+	@PostMapping("/instance")
+	public ComputeDTO getComputeInstance(@RequestBody ComputeDTO computeDTO)
 			throws IOException {
 
-		ComputeDTO response = computeService.getComputeInstance(instanceName);
+		ComputeDTO response = computeService.getComputeInstance(computeDTO.getInstanceName());
```

Comment:
What is the purpose of changing from Get to PostMapping when you are still getting the same parameter which is the instanceName?